src/nginxconfig/util/types_extensions.js

```diff
@@ -37,4 +37,5 @@ export const extensions = {
         'docx?|dotx?|docm|dotm|' +
         'xlsx?|xltx?|xlsm|xltm|' +
         'pptx?|potx?|pptm|potm|ppsx?',
+    html: 'html',
```

Comment:
```suggestion
    html: 'html?',
```
Allow `html` or `htm`